pygmt/src/inset.py

```diff
@@ -61,13 +61,13 @@ def inset(self, **kwargs):
         coordinates instead are the lower left and upper right corners of
         the desired rectangle. (Or, give *xmin/xmax/ymin/ymax* of bounding
         rectangle in projected coordinates and optionally
-        append **+u**\ *unit* [Default coordinate unit is meter (e)].
+        append **+u**\ *unit* [Default coordinate unit is meters (**e**)].
 
         Append **+w**\ *width*\ [/*height*] of bounding rectangle or box
-        in plot coordinates (inches, cm, etc.). By default, the anchor
-        point on the scale is assumed to be the bottom left corner (**BL**),
-        but this can be changed by appending **+j** followed by a 2-char
-        justification code *justify*.
+        in plot coordinates (inches, centimeters, etc.). By default, the
+        anchor point on the scale is assumed to be the bottom left corner
+        (**BL**), but this can be changed by appending **+j** followed by
+        a 2-char justification code *justify*.
```

Comment:
```suggestion
        a 2-character justification code *justify*.
```

Reply:
Good point @willschlitzer! Changed this in the commits https://github.com/GenericMappingTools/pygmt/pull/2278/commits/e1e402e6382e5b26042bdfe7c7c8e52381204ce6, https://github.com/GenericMappingTools/pygmt/pull/2278/commits/2d09a2a71b046670017735b8bcd0293540b69f3e, and https://github.com/GenericMappingTools/pygmt/pull/2278/commits/d4f46249f2abd4947eef400264a0723b74e32d28.